src/main/java/tds/common/data/mysql/spring/TdsPropertyRowMapper.java

```diff
@@ -2,19 +2,24 @@
 
 import org.springframework.beans.BeanWrapper;
 import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import tds.common.data.mysql.InstantPropertyEditorSupport;
 import tds.common.data.mysql.UuidPropertyEditorSupport;
 
+import java.time.Instant;
 import java.util.UUID;
 
+/**
+ * Handle MySQL-specific conversions of data stored in the TDS databases.
+ */
+public class TdsPropertyRowMapper<T> extends BeanPropertyRowMapper<T> {
 
-public class UuidBeanPropertyRowMapper<T> extends BeanPropertyRowMapper<T> {
-
-    public UuidBeanPropertyRowMapper(Class<T> type) {
+    public TdsPropertyRowMapper(Class<T> type) {
```

Comment:
What if we changed the constructor here to be `TdsPropertyRowMapper(Class<T> type, Pair<Class, PropertyEditorySupport> support...)`.  Then when the initBeanWrapper is called it can iterate over the array of Pairs and register the custom editors.  We can have a default constructor that just takes in class and has the UUID and the Instant editor already in an array.